packages/main/src/models/generative-model.test.ts

```diff
@@ -68,6 +81,15 @@ describe("GenerativeModel", () => {
       },
     );
     expect(genModel.generationConfig?.temperature).to.equal(0);
+    expect(genModel.generationConfig?.responseMimeType).to.equal(
+      "application/json",
+    );
+    expect(genModel.generationConfig?.responseSchema.type).to.equal(
+      FunctionDeclarationSchemaType.OBJECT,
+    );
+    expect(
+      genModel.generationConfig?.responseSchema.properties.testField.type,
+    ).to.equal(FunctionDeclarationSchemaType.STRING);
```

Comment:
I would like to add a test to ensure this is being passed to makeRequestStub and add this to the test below to ensure override works, as well as to the chat.sendMessage tests. I can make a PR to your branch to do it.

Reply:
Thank you! (addressed in #159)